### PR TITLE
Cross-Post https://cloud.google.com/blog/products/compute/serve-cloud-run-requests-with-grpc-not-just-http

### PR DESCRIPTION
**Pull request by gnossen**
I have not yet been able to inspect this post to see if it's reasonable due to https://github.com/grpc/grpc.io/issues/146, but I want to get the source up here.

CC @thisisnotapril

### CODE REVIEW

**Comment by gnossen** [comment]
I was able to verify that this looks good from the GitHub-based preview. This is now a very slick workflow. Not sure who to notify for a review. @chalin ?

File: content/blog/grpc-cloud-run.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+<!--more-->
+
+Most applications that run inside Cloud Run use HTTP JSON REST to serve requests, but that’s not the only protocol it supports; in September, it also started to support unary gRPC services.
+
+gRPC is a high performance RPC framework developed by Google and used extensively for traditional workloads and at the edge by companies like Netflix, Cisco, Square, and others. While gRPC offers advantages over traditional HTTP, like strong interface definitions and code generation, setting up the infrastructure to run a gRPC server in production can be a real chore. Cloud Run takes the toil out of this process.
+
+
+With gRPC, you start with a strong API contract in the form of a [protocol buffer file](https://developers.google.com/protocol-buffers?_ga=2.147498999.-1743553425.1584489606):
```
Comment (chalin) [inline content/blog/grpc-cloud-run.md]: Why the `?_ga=2.147498999.-1743553425.1584489606`?

File: content/blog/grpc-cloud-run.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: "Not just for HTTP anymore: gRPC comes to Cloud Run"
+date: 2020-03-17
+author:
+  name: Wenlei He
+  link: https://github.com/wlhee
+---
+
+*This is a cross-post of [an article](https://cloud.google.com/blog/products/compute/serve-cloud-run-requests-with-grpc-not-just-http)
```
Comment (chalin) [inline content/blog/grpc-cloud-run.md]: If this is just a cross-post, why distract the reader. I'd suggest moving this to the end of the page, something like this:

> ...
> _Google Cloud blog [cross-post](https://cloud.google.com/blog/products/compute/serve-cloud-run-requests-with-grpc-not-just-http)._

File: content/blog/grpc-cloud-run.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      logging.info("Received request: %s", request)
+      if request.operation == calculator_pb2.ADD:
+          result = request.first_operand + request.second_operand
+      else:
+          result = request.first_operand - request.second_operand
+      return calculator_pb2.CalculationResult(result=result)
+```
+Cloud Run provides everything else you need to get your code serving traffic. You just need to put together a simple Dockerfile and run a few commands:
+
+```
```
Comment (chalin) [inline content/blog/grpc-cloud-run.md]: Use `sh` as a language here?

File: content/blog/grpc-cloud-run.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+---
+title: "Not just for HTTP anymore: gRPC comes to Cloud Run"
+date: 2020-03-17
+author:
+  name: Wenlei He
+  link: https://github.com/wlhee
```
Comment (chalin) [inline content/blog/grpc-cloud-run.md]: Maybe add the multiple authors back and the authors will show up once #148 is fixed?

File: content/blog/grpc-cloud-run.md (hunk 1 above)
Comment (gnossen) [reply content/blog/grpc-cloud-run.md]: It's a good question. I copied it verbatim from the original article. There's almost certainly no harm in removing it.

**Comment by gnossen** [comment]
@chalin PTALA

**Comment by gnossen** [comment]
Sorry. Unreliable WFH wireless mouse misfired. I didn't actually intend to close the PR. 😅

**Comment by gnossen** [comment]
@lucperkins I think this is ready for merge. Thank you for the review!

**Comment by chalin** [comment]
Merged.

**Pull request merged**